src/backend/expungeservice/endpoints/auth.py

```diff
@@ -94,23 +19,34 @@ def post(self):
 
         check_data_fields(data, ['email', 'password'])
 
-        user_db_result = user.read(
+        user_db_result = user_db_util.read(
             g.database,
-            user.identify_by_email(g.database, data['email']))
+            user_db_util.identify_by_email(g.database, data['email']))
 
         if (not user_db_result or
                 not check_password_hash(user_db_result['hashed_password'],
                                         data['password'])):
             error(401, 'Invalid username or password')
 
-        response_data = {
-            'auth_token': get_auth_token(current_app,
-                                         user_db_result['user_id']),
-            'user_id': user_db_result['user_id']
-        }
-        return jsonify(response_data)
+        user = from_dict(data_class=User, data=user_db_result)
+        User.login_user(user)
+        return jsonify({})
```

Comment:
The frontend still needs to obtain and store the logged in user’s id to read or update the user’s own profile info. Shouldn’t that still be here?

Reply:
Ah yes I have yet to remove the user ids from the `/api/users` backend endpoint. I'll follow up with some commits on this PR soon.

Reply:
I've added a new endpoint `/api/user` (`GET`, `POST`) to handle the respective requests for the current user. Please see the new commit and commit message for more information.